### PR TITLE
Adds internationalization for the word "here". Issue #702

### LINKED ISSUE #702: the "here" for help links not internationalized
### Is your feature request related to a problem? Please describe.

Most web links have a "here" link. The word here should be internationalized.

### Describe the solution you'd like

src/client/app/components/TooltipHelpComponentAlternative.tsx has the component for this. The word "here" is inside the return in the loop about links. This is what needs to be addressed.

### Describe alternatives you've considered

None

### Additional context

None

### PR DESCRIPTION
**Pull request by jbmayor4**
# Description

This pull request addresses the bug found in Issue #702. It adds internalization support similar to other areas of OED for the word "here", when it is in reference to a link. This change affects the translation "data.js" as well as the toolTipHelpComponentAlternative for generating help links.

Fixes #702 

## Type of change

(Check the ones that apply by placing an "x" instead of the space in the [ ] so it becomes [x])

- [ ] Note merging this changes the node modules
- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

(Note what you have done by placing an "x" instead of the space in the [ ] so it becomes [x]. It is hoped you do all of them.)

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request

## Limitations

Issue should be fixed. I do not believe there is any further work to be done on this issue.

### CODE REVIEW

**Comment by huss** [comment]
Hi @jbmayor4 and thank you for your PR. I don't see that you have signed our CLA. If you think you did then let me know. I did see a notice of a new signature but I did not see yours. We need a CLA to accept a contribution from someone. If you have questions then please let the project know.

**Comment by jbmayor4** [comment]
Hi @huss , I thought I signed the CLA, but in case I am wrong I have just resigned it. Let me know if there is anything else you need on my end.

**Comment by huss** [comment]
Hi @jbmayor4. So, now I see both of your responses. It may have been a sync problem (or something I did wrong) but it clearly was not your issue as it shows you did it before and now. I appreciate your quick response. I'll try to work on this PR soon. Thank you for your first contribution.

File: src/client/app/translations/data.js [hunk 1]
```diff
@@ -132,6 +132,7 @@ const localeData = {
 		"hide": "Hide",
 		"hide.options": "Hide options",
 		"hide.options.in.link": "Hide menu in link",
+		"here": "here",
```
Comment (huss) [inline src/client/app/translations/data.js]: This is slightly off for where it should be alphabetically. It should be before "hide".

File: src/scripts/databaseBackup.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (huss) [inline src/scripts/databaseBackup.sh]: This file does not relate to this PR but another one. I think it should be removed from this PR.

**Review by huss — CHANGES_REQUESTED** [review]
Thanks to @jbmayor4 for addressing this issue. The code works fine. I have made a couple of (hopefully) quick change requests. We appreciate your first contribution.

**Comment by huss** [comment]
It has been about 2 weeks weeks since the PR review. I wanted to know if @jbmayor4  plans to make these changes or not. It is okay either way it would be nice to know so the project can continue to resolve this PR. There is also a merge conflict from new code on Tooltips that I can take care of once the other items are fixed up.

File: src/client/app/translations/data.js (hunk 1 above)
Comment (huss) [reply src/client/app/translations/data.js]: I'm fixed this.

File: src/scripts/databaseBackup.sh (hunk 2 above)
Comment (huss) [reply src/scripts/databaseBackup.sh]: I'm removing this file.

**Review by huss — APPROVED** [review]

**Comment by huss** [comment]
Since the project did not hear from @jbmayor4 in two weeks, I have made the requested changes and resolved the merge conflict. The PR is now ready to merge.

**Pull request merged**